canopen/common.py

```diff
@@ -63,8 +67,8 @@ def raw(self):
         """
         value = self.od.decode_raw(self.data)
         text = "Value of %s (0x%X:%d) is %r" % (
-            self.od.name, self.od.index,
```

Comment:
Could you change to use self.name here and below?

Reply:
Sorry, I missed that in #82.  Should I push it there or just include the change here?

Reply:
I merged the other one so you can do it here while merging with master.

Reply:
Made a new PR #84.  I don't know how I could fix it in this PR, because you will do the merging and the `common.Variable.name` changes are not included in (nor related to) this branch.